UnityProject/Assets/Scripts/Core/Transform/ScaleSync.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using Mirror;
+using UnityEngine;
+using UnityEngine.Networking;
+
+
+namespace Scripts.Core.Transform
+{
+
+    public class ScaleSync : NetworkBehaviour
```

Comment:
Where did you put this on the explosive, on the sprite object or the top level object?

Reply:
On the sprite itself.

Reply:
You'll need to move it to the parent and then select the wanted transform in the editor as having it on the sprite will add a network identity to the child object.

Reply:
Done.